components/email/client/editorial_signup.coffee

```diff
@@ -80,6 +84,32 @@ module.exports = class EditorialSignupView extends Backbone.View
       error: ->
         cb null
 
+  showEditorialCTA: ->
+    @test = splitTest('editorial_cta_banner')
+    if @test.outcome() is 'banner'
```

Comment:
It might be cleaner to extract this out into a variable, like `outcome = @test.outcome()`. It'd be a performance gain (avoids calling it an additional 6 times) since `outcome()` is actually reading the Cookie and retrieving the outcome from it.